### DISCUSSION TITLE
Official IRC or Matrix

### DISCUSSION THREAD

**Discussion by alien2003** [R2Northstar/Northstar#163] [General]
I can see official Discord in description but not everyone agrees to trade their privacy for their favorite game so it would be great to also have official IRC or Matrix room, maybe bridged to Discord

**Comment by Agrigohar** [comment]
Well I think that is pretty ironic to say, because you are already trading your privacy with Microsoft because you use GitHub. And if you see what security flaws the Titanfall Software has, you might also trade stuff like your IP with anyone who has a bit of IT knowledge.

**Comment by ghost** [comment]
>And if you see what security flaws the Titanfall Software has, you might also trade stuff like your IP with anyone who has a bit of IT knowledge.

VPN 🤦🏻

**Comment by elterrasense** [comment]
> 
> VPN 🤦🏻

Yeah because VPNs clearly work and they don't steal your data instead.
I could rant for hours about it and how no matter what you do nowadays there's always something spying on you, but it would be going overboard for just a github issue lol.
Anyways, having a bit of sense of privacy like this user is good, but it's no use being an schizo about it, at that point you might as well not use the internet at all.

**Comment by ghost** [comment]
There are fully open-source VPNs which can be bought with crypto and don't need email, etc. Also there's an option of self hosting. I prefer to trust my VPN provider and don't get my IP exposed to some random out there.

**Comment by 0x14307** [comment]
> 

this ain't GTA but I see your point.

Plus most people who play games wouldn't use IRC or Matrix anyways

**Comment by alien2003** [comment]
> Well I think that is pretty ironic to say, because you are already trading your privacy with Microsoft because you use GitHub. And if you see what security flaws the Titanfall Software has, you might also trade stuff like your IP with anyone who has a bit of IT knowledge.

Discord is known for crawling files and scanning process list on your computer. Titanfall 2 was not catched doing stuff like that. Github also can't access data on your computer

**Comment by 0x14307** [comment]
> > Well I think that is pretty ironic to say, because you are already trading your privacy with Microsoft because you use GitHub. And if you see what security flaws the Titanfall Software has, you might also trade stuff like your IP with anyone who has a bit of IT knowledge.

Sources?

**Comment by alien2003** [comment]
> source?

https://spyware.neocities.org/articles/discord.html

If you don't trust this source, you can do your own research by running gdb~W Process Monitor

**Reply by ScalyChimp** [reply]
> https://spyware.neocities.org/articles/discord.html
> 
> If you don't trust this source, you can do your own research by running gdb~W Process Monitor

This website also says steam is extremely likely to be spyware, and unless you're using origin and want to argue that origin is more respectful of privacy than steam, it doesn't seems like there's much point to avoiding discord if you've already installed steam/origin to buy and play TF|2.

I think someone said this earlier, but if you care that much about this stuff, you should really just avoid the internet.

**Comment by 0x14307** [comment]
>neocities

**Comment by GeckoEidechse** [comment]
Fuck it, I just made one now on matrix.org :P
https://matrix.to/#/#r2northstar:matrix.org

Doesn't have bridging or anything yet, if someone's dedicated enough feel free to get in touch with admins of the Discord server to set things up.

**Comment by zImPatrick** [comment]
> Discord is known for crawling files and scanning process list on your computer. Titanfall 2 was not catched doing stuff like that. Github also can't access data on your computer

Couldn't you just use discord web?

**Comment by alien2003** [comment]
> Couldn't you just use discord web?

It still requires phone number to use, not good for FOSS projects

**Reply by barnabwhy** [reply]
Bruh I never had to put my phone number in. What are you on about?

**Reply by barnabwhy** [reply]
It requires your email address and that's it, unless they're asking for phone verification nowadays.

**Reply by barnabwhy** [reply]
![image](https://user-images.githubusercontent.com/22575741/150598542-9d7a5ee5-87de-4a83-8ab6-0b898dfa95d8.png)

**Comment by MysteriousRSA** [comment]
My guy, we as a society have gotten to the point where if you have internet, your ISP, government, phone manufacturer, and everything you use on the internet already know almost everything.

You're phone tracks everything you do already, you already gave your credit card info and email to EA, who already sold it probably, discord is, at least in my opinion, much more reliable than EA
